### PR TITLE
Update Russian language

### PR DESCRIPTION
**Pull request by WebVPF**

### CODE REVIEW

**Comment by mjauvin** [comment]
@webeks did you translate these yourself? You do speak russian, right?

**Comment by bennothommo** [comment]
@WebVPF I recognise that it's not always accurate, but Google Translate is saying that all your translations are to "suspend" the user, whereas the messages should be when you are "unsuspending" a user. Can you confirm these translations are correct?

**Comment by bennothommo** [comment]
For example, "Пользователь был приостановлен." is being translated to "The user has been suspended."

File: lang/ru/lang.php [hunk 1]
```diff
@@ -63,6 +63,9 @@
         'unban_selected_confirm' => 'Разблокировать выбранных пользователей?',
         'unban_selected_empty' => 'Нет выбранных пользователей для разблокировки.',
         'unban_selected_success' => 'Выбранные пользователи успешно разблокированы.',
+        'unsuspend' => 'Приостановлен',
```
Comment (idevmans) [inline lang/ru/lang.php:66]: 'unsuspend' => 'Возабновлён',

File: lang/ru/lang.php [hunk 2]
```diff
@@ -63,6 +63,9 @@
...
+        'unsuspend_success' => 'Пользователь был приостановлен.',
```
Comment (idevmans) [inline lang/ru/lang.php:67]: 'unsuspend_success' => 'Пользователь был возабновлён.',

File: lang/ru/lang.php [hunk 3]
```diff
@@ -63,6 +63,9 @@
...
+        'unsuspend_confirm' => 'Приостановить данного пользователя?',
```
Comment (idevmans) [inline lang/ru/lang.php:68]: 'unsuspend_confirm' => 'Возобновить данного пользователя?',

**Review by idevmans — COMMENTED** [review]
gramma fix

File: lang/ru/lang.php (hunk 3 above)
Comment (mjauvin) [reply lang/ru/lang.php:68]: which one is it?

Comment (WebVPF) [reply lang/ru/lang.php]: Thank you! Fixed

Comment (idevmans) [reply lang/ru/lang.php]: > which one is it?

Everything, but he already fixed it

**Pull request merged**